assets/src/scss/blocks/PostsListBlock.scss

```diff
@@ -0,0 +1,143 @@
+.posts-list-query {
+  .wp-block-post-template.is-layout-flow li {
+    width: 100% !important;
+  }
+
+  .wp-block-columns.is-layout-flex {
+    gap: 20px;
+  }
+
+  .wp-block-post-featured-image {
+    flex-basis: 30%;
+
+    img {
+      @include medium-and-up {
+        height: 180px;
+      }
+
+      @include large-and-up {
+        height: 210px;
+      }
+      width: 100% !important;
+      object-fit: cover;
+    }
+  }
+
+  .wp-block-group.is-layout-flow {
+    flex-basis: 70%;
+  }
+
+  .wp-block-post-title {
+    font-weight: 700;
+    color: var(--grey-900);
+    font-family: var(--font-family-tertiary);
+    font-size: 1.375rem;
+    margin-top: 5px;
```

Comment:
Can we use spacing variables? i.e. `$sp-1x` which is equal to `6px` instead of `5px`. The we have all the spacing values already standardized.